client/components/quizeditor/CreatedQuizHistory.jsx

```diff
@@ -81,7 +81,7 @@ const CreatedQuizHistory = ({ loggedInUser, friends }) => {
         />
       </ul>
       {questionsLoaded
-        ? <QuizEditor questions={questionsToEdit} quizName={quizName} handleRenderingQuestions={handleRenderingQuestions} returnQuizList = {returnQuizList} editorClassName = {editorClassName}/>
+        ? <QuizEditor questions={questionsToEdit} quizName={quizName} handleRenderingQuestions={handleRenderingQuestions} returnQuizList = {returnQuizList} editorClassName = {editorClassName} userId={loggedInUser.id}/>
```

Comment:
Can you break this line up so it's a little easier to read? By that, I just mean can you spread it out over a couple of indented lines so that there isn't any side scrolling required to look at the code?